src/prop.ts

```diff
@@ -218,12 +218,15 @@ function baseProp(
   }
 
   if (whatis === WhatIsIt.ARRAY) {
+
+    const virtualSchemaItem = _buildSchema(Type,  { _id: rawOptions._id });
```

Comment:
why is this change needed?

Reply:
To support Inheritance for items in arrays. This line is necessary to pass the test: https://github.com/hasezoey/typegoose/pull/21/files#diff-ea520afea15a8887a0395e188d12c020R38